e2e/services/EventStream.test.ts

```diff
@@ -30,10 +28,12 @@ describe('EventStream', () => {
   };
 
   it('should work on http', async () => {
-    await startEventStream('http://176.9.63.35:9999/events/main');
+    // TODO Enable event stream on GitHub CI
```

Comment:
Is there a ticket number associated with this TODO? If so, would you mind adding it as part of the comment, please?

Reply:
I think this should be done in the separate PR since this PR intend to add browser test :).
I will create an issue indicating this